packages/paste-website/src/pages/foundations/typography/index.mdx

```diff
@@ -0,0 +1,692 @@
+
+<Card>
+  <TextColorSwatches />
+</Card>
+<br />
+
+## Our components
+
+We know how time-consuming it is to fiddle with type styles to get the correct one. This is why we have created type components, so they have style built into them. You can focus on designing full layouts, and we can worry about the nitty-gritty.
+
+<Card>
+  <DisplayHeading as="div" variant="displayHeading10" marginBottom="space0">
+    DisplayHeading10
+  </DisplayHeading>
+  <br />
+  <InlineCode>Font-family: Twilio Sans Display</InlineCode>
+  <br />
+  <InlineCode>Font-weight: extrabold</InlineCode>
+  <br />
+  <InlineCode>Font-size: 64px</InlineCode>
+  <br />
+  <InlineCode>Line-height: 80px</InlineCode>
+  <br />
+  <InlineCode>Text-color: color-text</InlineCode>
+  <br />
+  <Separator orientation="horizontal" verticalSpacing="space40" />
+  <Heading as="div" variant="heading10" marginBottom="space0">
+    Heading10
+  </Heading>
+  <br />
+  <InlineCode>Font-family: Twilio Sans Text</InlineCode>
+  <br />
+  <InlineCode>Font-weight: semibold</InlineCode>
+  <br />
+  <InlineCode>Font-size: 32px</InlineCode>
+  <br />
+  <InlineCode>Line-height: 44px</InlineCode>
+  <br />
+  <InlineCode>Letter-spacing: -.02rem</InlineCode>
+  <br />
+  <InlineCode>Text-color: color-text</InlineCode>
+  <br />
+  <Separator orientation="horizontal" verticalSpacing="space40" />
+  Paragraph
+  <br />
+  <br />
+  <InlineCode>Font-family: Twilio Sans Text</InlineCode>
+  <br />
+  <InlineCode>Font-weight: normal</InlineCode>
+  <br />
+  <InlineCode>Font-size: 14px</InlineCode>
+  <br />
+  <InlineCode>Line-height: 24px</InlineCode>
+  <br />
+  <InlineCode>Text-color: color-text</InlineCode>
+  <br />
+  <Separator orientation="horizontal" verticalSpacing="space40" />
+  <DetailText>Detail Text</DetailText>
+  <br />
+  <InlineCode>Font-family: Twilio Sans Text</InlineCode>
+  <br />
+  <InlineCode>Font-weight: medium</InlineCode>
+  <br />
+  <InlineCode>Font-size: 12px</InlineCode>
+  <br />
+  <InlineCode>Line-height: 20px</InlineCode>
+  <br />
+  <InlineCode>Text-color: color-text-weak</InlineCode>
+  <br />
+</Card>
+<br />
+
+Here are some of our typography components:
+
+- [Anchor](/components/anchor)
+- [Description List](/components/description-list)
+- [Detail Text](/components/detail-text)
+- [Display Heading](/components/display-heading)
+- [Heading](/components/) -[Inline Code](/components/heading)
+- [Label](/components/label)
+- [Help Text](/components/help-text)
+- [List](/components/list)
+- [Paragraph](/components/paragraph)
+- [Truncate](/components/truncate)
+
+Most of our block-level typography components like Heading, Paragraph, and Detail Text have preset margins so that stacking components will give your layout the ideal amount of spacing without having to adjust margins manually too much yourself.
+
+<LivePreview scope={{Heading, Paragraph, Card}} language="jsx" showOverflow>
+  {`<Card padding="space0">
```

Comment:
question: was the lack of padding around the card content intentional?

Reply:
yes - to show the bottom margin difference but lmk if you think of a better way

Reply:
([reference to example](https://docs.google.com/document/d/10EKRuDy0fO90IxYOrJ2vUm2d4ijHqXsgfqgUVWd_XyM/edit?disco=AAAArEsssHQ))